saskatoon/sitebase/templates/registration/change_password.html

```diff
@@ -0,0 +1,65 @@
+{% extends 'app/base/view_empty.html' %}
+{% load i18n %}
+{% load static %}
+{% block body %}
+    <style>
+      li::before {
+        content: "• ";
+      }
+    </style>
+    <div class="login-content">
+        <!-- Login -->
+        <div class="nk-block toggled" id="l-login">
+            <form action='{% url "change_password" %}' method="post">
+                {% csrf_token %}
+                <div class="nk-form">
+                    <h1>Change password</h1>
+                    {% block form %}
```

Comment:
Are the `{% block form %}` tags doing anything here?  
in `apps/forms/base.html` for example, they are used as placeholders so other templates extending it can define the block. I don't think this is what you are doing here
https://github.com/LesFruitsDefendus/saskatoon-ng/blob/9eee8ffba13f0978916fab800ec3970be38a8737/saskatoon/sitebase/templates/app/forms/base.html#L33

Reply:
definitely not. i think i just copied existing login form. ill clean that one up too.